evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -128,9 +128,9 @@ def test_all_estimators(
         assert len((_all_estimators_used_in_search())) == 9
     else:
         if is_using_conda:
-            n_estimators = 15
+            n_estimators = 14
```

Comment:
Why is only -1?

Reply:
Hmm, it looks like we actually don't need to change this actually since both implementations were excluded as part of `_not_used_in_automl`. Thus, removing the sklearn versions actually doesn't have an impact.